include/boost/math/special_functions/fourier_transform_daubechies_scaling.hpp

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright Nick Thompson, Matt Borland, 2023
+ * Use, modification and distribution are subject to the
+ * Boost Software License, Version 1.0. (See accompanying file
+ * LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+ */
+
+#ifndef BOOST_MATH_SPECIAL_FOURIER_TRANSFORM_DAUBECHIES_SCALING_HPP
+#define BOOST_MATH_SPECIAL_FOURIER_TRANSFORM_DAUBECHIES_SCALING_HPP
+#include <array>
+#include <cmath>
+#include <complex>
+#include <iostream>
+#include <limits>
+#include <boost/math/constants/constants.hpp>
+#include <boost/math/tools/big_constant.hpp>
+#include <boost/math/tools/estrin.hpp>
+
+namespace boost::math {
+
+namespace detail {
+
+// See the Table 6.2 of Daubechies, Ten Lectures on Wavelets.
+// These constants are precisely those divided by 1/sqrt(2), because otherwise
+// we'd immediately just have to divide through by 1/sqrt(2).
+// These numbers agree with Table 6.2, but are generated via example/calculate_fourier_transform_daubechies_constants.cpp
+template <typename Real, unsigned N> constexpr const std::array<Real, N> ft_daubechies_scaling_polynomial_coefficients() {
```

Comment:
```suggestion
template <typename Real, unsigned N> constexpr std::array<Real, N> ft_daubechies_scaling_polynomial_coefficients() {
```

`constexpr const` is redundant from C++14.